news/templates/news/article_detail.html

```diff
@@ -4,5 +4,21 @@
 <h1>{{ object.title }}</h1>
 <p>{{ object.published }}</p>
 <p>{{ object.content | safe }}</p>
+<div>		
+	<div>
+			<form action="/article/{{object.slug}}/comment" method="post">
```

Comment:
remove hardcoded action url and use django's template syntax, e.g.:
```
# urls.py
urlpatterns += patterns('myview.views',
    url(r'^(?P<user>\w+)/', 'myview', name='myurl'), # I can't think of a better name
)

# template.html
<form name="form" method="post" action="{% url myurl username %}">
```
assuming you have a variable _username_ in your template